README.md

```diff
@@ -1,240 +1,20 @@
 # 🧠🤖Deep Agents
 
 Using an LLM to call tools in a loop is the simplest form of an agent. 
-This architecture, however, can yield agents that are “shallow” and fail to plan and act over longer, more complex tasks. 
-Applications like “Deep Research”, "Manus", and “Claude Code” have gotten around this limitation by implementing a combination of four things:
+This architecture, however, can yield agents that are "shallow" and fail to plan and act over longer, more complex tasks. 
+Applications like "Deep Research", "Manus", and "Claude Code" have gotten around this limitation by implementing a combination of four things:
 a **planning tool**, **sub agents**, access to a **file system**, and a **detailed prompt**.
 
 <img src="deep_agents.png" alt="deep agent" width="600"/>
 
 `deepagents` is a Python package that implements these in a general purpose way so that you can easily create a Deep Agent for your application.
 
-**Acknowledgements: This project was primarily inspired by Claude Code, and initially was largely an attempt to see what made Claude Code general purpose, and make it even more so.**
-
 ## Installation
 
 ```bash
 pip install deepagents
 ```
 
-## Usage
-
-(To run the example below, will need to `pip install tavily-python`)
-
-```python
-import os
-from typing import Literal
-
-from tavily import TavilyClient
-from deepagents import create_deep_agent
-
-
-# Search tool to use to do research
-def internet_search(
-    query: str,
-    max_results: int = 5,
-    topic: Literal["general", "news", "finance"] = "general",
-    include_raw_content: bool = False,
-):
-    """Run a web search"""
-    tavily_async_client = TavilyClient(api_key=os.environ["TAVILY_API_KEY"])
-    return tavily_async_client.search(
-        query,
-        max_results=max_results,
-        include_raw_content=include_raw_content,
-        topic=topic,
-    )
-
-
-# Prompt prefix to steer the agent to be an expert researcher
-research_instructions = """You are an expert researcher. Your job is to conduct thorough research, and then write a polished report.
-
-You have access to a few tools.
-
-## `internet_search`
-
-Use this to run an internet search for a given query. You can specify the number of results, the topic, and whether raw content should be included.
-"""
-
-# Create the agent
-agent = create_deep_agent(
-    [internet_search],
-    research_instructions,
-)
-
-# Invoke the agent
-result = agent.invoke({"messages": [{"role": "user", "content": "what is langgraph?"}]})
-```
-
-See [examples/research/research_agent.py](examples/research/research_agent.py) for a more complex example.
-
-The agent created with `create_deep_agent` is just a LangGraph graph - so you can interact with it (streaming, human-in-the-loop, memory, studio)
-in the same way you would any LangGraph agent.
```

Comment:
can we leave this as a "quckstart"? I think having that in the readme makes it way easier to get started